From 43d12643185887c95ebc95e06182369a727014a1 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Tue, 7 Jan 2014 12:10:52 -0500
Subject: [PATCH 1272/1825] a38x: Fix PHY access issue

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 4dc968a9b071f198e9124120332f278b7c44e17b

	- Found on bonding testing
	- Add missed mvSysEthPhyInit() function call

Change-Id: I30cdefe9ba33cbb82383e6ac118f8fe3e3be7100
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/5034
Reviewed-by: Uri Eliyahu <uriel@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-armada38x/core.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-armada38x/core.c b/arch/arm/mach-armada38x/core.c
index 4330f1d..8e67ca5 100644
--- a/arch/arm/mach-armada38x/core.c
+++ b/arch/arm/mach-armada38x/core.c
@@ -672,6 +672,7 @@ static void __init eth_init(void)
 void __init a38x_init_eth(void)
 {
 #ifdef CONFIG_MV_ETHERNET
+	mvSysEthPhyInit();
 	eth_init();
 #endif
 }
-- 
1.7.5.4

